packages/hub/routes/checkly-webhook.ts

```diff
@@ -0,0 +1,52 @@
+import Koa from 'koa';
+import autoBind from 'auto-bind';
+import { inject } from '@cardstack/di';
+import StatuspageApi from '../services/statuspage-api';
+
+type CheckName = 'hub-prod subgraph / RPC node block number diff within threshold';
+type Checks = {
+  [key in CheckName]: {
+    componentName: string; // Component name in Checkly
+    incidentMessage: string; // Will be shown in Statuspage
+  };
+};
+
+export default class ChecklyWebhookRoute {
+  statuspageApi: StatuspageApi = inject('statuspage-api', { as: 'statuspageApi' });
+
+  checks: Checks = {
+    'hub-prod subgraph / RPC node block number diff within threshold': {
+      componentName: 'Subgraph',
+      incidentMessage: 'Subgraph block number is behind RPC block number by more than 10 blocks',
```

Comment:
Should this `10` come from config or a constant?

Reply:
i think it already does, but that was part of a different PR that monitors the subgraph block height delta. this PR is more focused on opening and closing incidents in status page.

Reply:
Should we adjust this message to be a bit more user friendly, and explain the possible consequences? 

The following is probably not the best way to describe this situation, but I think something like that would be more suitable. Can you help to form a better message?

For example: "We're experiencing issues with lag in the sync between the subgraph and the chain. This can result in increased time needed for transactions."

Reply:
Sure, may something like "Experiencing blockchain indexing delay: the blockchain index delayed by at least 10 blocks". But like @lukemelia was saying instead of hard coding `10`, let's export `DEGRADED_THRESHOLD` from the `StatusRoute` and use that variable in the message.